redhat-cop/casl-ansible
# add wildcard record to Private DNS

#### What does this PR do?
Adds the DNS wildcard to the private DNS view

#### How should this be manually tested?
1) From the app-0 VM instance execute:   dig @<`private IP of dns vm`>  <`env_id.dns_domain`> axfr
verify:  *.apps.<`dns_domain`> is pointing to IP of  `infranode`

#### Is there a relevant Issue open for this?
https://github.com/redhat-cop/casl-ansible/issues/52

#### Who would you like to review this?
cc: @oybed
